doc/man1/flux-kvs.adoc

```diff
@@ -42,13 +42,21 @@ arguments are described below.
 
 COMMANDS
 --------
-*get* 'key' ['key...']::
+*get* [-j|-r|-t] [-a treeobj] 'key' ['key...']::
 Retrieve the value stored under 'key'.  If nothing has been stored under
-'key', display an error message.
+'key', display an error message.  If no options, value is interpreted
+as a NULL-terminated string.  If '-j', it is interpreted as encoded JSON.
+If '-r', it is interpreted as raw data and is output without formatting.
+If '-t', the RFC 11 object is displayed.  '-a treeobj' causes the lookup
+to be relative to an RFC 11 snapshot reference.
 
-*put* 'key=value' ['key=value...']::
+*put* [-j|-r] 'key=value' ['key=value...']::
```

Comment:
need -t and -n

Reply:
I could swear I already did that one.  Thanks!